Source/SwiftLintFramework/Extensions/Array+SwiftLint.swift

```diff
@@ -45,11 +45,11 @@ extension Array {
             return (copy[0..<pivot], copy[pivot..<count])
     }
 
-    func parallelFlatMap<T>(transform: @escaping ((Element) -> [T])) -> [T] {
+    func parallelFlatMap<T>(transform: (Element) -> [T]) -> [T] {
```

Comment:
I was about to say this is wrong, but then after thinking about it some more realized that it was _previously_ wrong! Thanks! 😅

Reply:
Hah, no worries 😁 Yeah if I understand closure allocation correctly this should improve performance / reduce copying of the linters that get passed around.